locales/en/plugin__odf-console.json

```
@@ -22,11 +22,16 @@
   "Checkbox to select cluster": "Checkbox to select cluster",
   "Data policies": "Data policies",
   "Disaster recovery": "Disaster recovery",
+  "Application name": "Application name",
+  "application name search": "application name search",
+  "No matching application found": "No matching application found",
+  "Apply DRPolicy": "Apply DRPolicy",
+  "Select the Applications for applying the DRPolicy": "Select the Applications for applying the DRPolicy",
+  "Disaster recovery(DR) protection will be applied for all the PersistentVolumeClaims(PVCs) under the selected Applications.": "Disaster recovery(DR) protection will be applied for all the PersistentVolumeClaims(PVCs) under the selected Applications.",
```

```suggestion
  "Disaster recovery(DR) protection will be applied for all the PersistentVolumeClaims(PVCs) under the selected Applications.": "Disaster recovery(DR) protection will be applied for all the persistent volume claims(PVCs) under the selected Applications.",
```

---

This is CRD name, as per capitalization we need to use like this

---

Its a general reference.
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references

---

Also Disaster recovery(DR) protection -> Disaster recovery(DR)  ? Protection is well understood by Disaster recovery(DR) ?
```suggestion
  "Disaster recovery(DR) protection will be applied for all the PersistentVolumeClaims(PVCs) under the selected Applications.": "Disaster recovery(DR) protection will be applied to all the PersistentVolumeClaims(PVCs) under the selected Applications.",
```

---

i discussed this with UX team and they feel like:

Disaster recovery is a process/practice and we don't necessarily apply a process to anything it is rather the protection that is enabled for the PVCs.

---

also, i changed Application to application where ever it not mentioned (CRD) specifically

---

> Its a general reference. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references

Got it